src/main/java/no/unit/nva/cristin/projects/CristinApiClient.java

```diff
@@ -94,9 +75,15 @@ public ProjectsWrapper queryCristinProjectsIntoWrapperObjectWithAdditionalMetada
         Map<String, String> requestQueryParams) throws ApiGatewayException {
 
         long startRequestTime = System.currentTimeMillis();
-        List<NvaProject> nvaProjects = importProjectsFromCristin(requestQueryParams);
+        HttpResponse<InputStream> response = queryProjects(requestQueryParams);
+        List<CristinProject> enrichedProjectsFromResponse =
```

Comment:
What makes it an enriched project?

Maybe call this mapResponseToCristinProjects()

Reply:
Query of projects does not give us much data about each project, so we have to manually lookup each project from the query using id to get more fields